include/wx/gtk/private/addremovectrl.h

```diff
@@ -38,7 +38,9 @@ class wxAddRemoveImpl : public wxAddRemoveImplBase
         GtkToolbar* const toolbar = m_tbar->GTKGetToolbar();
         GtkStyleContext* context = gtk_widget_get_style_context(GTK_WIDGET(toolbar));
         gtk_style_context_add_class(context, GTK_STYLE_CLASS_INLINE_TOOLBAR);
+#ifndef __WXGTK4__
         gtk_style_context_set_junction_sides(context, GTK_JUNCTION_TOP);
```

Comment:
Do things still look acceptably without this? Could you please check the "add/remove" dialog in the dialogs sample?

It's important to fix compilation, of course, but I'd like to avoid breaking the run-time appearance too...

Reply:
I'll just drop this change for now and leave a comment, someone else can look at it when you get there, this is from a different compilation path.